builder/googlecompute/artifact.go

```diff
@@ -140,8 +140,6 @@ func (a *Artifact) State(name string) interface{} {
 		return a.image.Name
 	case "ImageSizeGb":
 		return a.image.SizeGb
-	case "AccountFilePath":
```

Comment:
Nice! This removal needs to be added in the CHANGELOG to denote that the artifact response has changed.

Reply:
Yes! We should also call-out that with this gone, the `googlecompute-export` post-processor will need its own set of credentials, as it will not harvest it from the artifact after this change.